book/src/html-macro/special-attributes/README.md

```diff
@@ -0,0 +1,55 @@
+# Special Attributes
```

Comment:
These docs are very technical.

That's fine. Technical details can be useful for a technical audience.

But, more importantly, these docs should be designed around:

- reader should quickly understand whether they need to care
- if they do need to care, reader should quickly understand what aspects they need to care about

A reader is either coming here because:

- they're casually browsing the docs
- they ran into a problem that they're trying to figure out how to solve

Casual browser just wants a quick sense of why any of this matters.
Person with a problem wants to quickly find a solution.

We can continue to infuse technical details since they arm the reader with foundational knowledge that, as it accumulates, will help them write better applications.
However, we need to be sure that the technical details don't overshadow the typical user's main need, which is usually to quickly sift through information until they find what they're looking for.

## Recommendations

- In the `## checked Attribute` section, start with something like:
    ```
    According to the HTML spec, the "checked" attribute only controls the default checkedness.
    Changing the "checked" attribute may not cause the checkbox's checkedness to change.
    
    When setting the "checked" attribute via the `html!` macro, `percy` will also update the
    input element's "checkedness".
    
    If they change from `html! { input checked=true /> }` to `html { <input checked=false /> }`, the input
    elements checkedness will definitely change.
    
    This behavior is more desirable because `percy` developers are accustomed to declaratively controlling the DOM.
    ```

- Can use a similar explanation for the `value` attribute's special behavior.